This allows us to access some function from wchar.h that other platforms
can get from stdio.h.
Potentially send upstream with an ifdef Solaris statement.

--- comm-esr31/mozilla/intl/hyphenation/src/hnjalloc.h.orig	2015-06-04 17:42:51.598696275 -0700
+++ comm-esr31/mozilla/intl/hyphenation/src/hnjalloc.h	2015-06-04 17:42:51.603672670 -0700
@@ -24,6 +24,7 @@
  */
 
 #include <stdio.h> /* ensure stdio.h is loaded before our macros */
+#include <wchar.h>
 
 #undef FILE
 #define FILE hnjFile
